cmd/kubeadm/app/cmd/upgrade/common.go

```diff
@@ -76,7 +72,7 @@ func getK8sVersionFromUserInput(flags *applyPlanFlags, args []string, versionIsM
 
 // enforceRequirements verifies that it's okay to upgrade and then returns the variables needed for the rest of the procedure
 func enforceRequirements(flags *applyPlanFlags, dryRun bool, newK8sVersion string) (clientset.Interface, upgrade.VersionGetter, *kubeadmapi.InitConfiguration, error) {
-	client, err := getClient(flags.kubeConfigPath, dryRun)
+	client, err := upgrade.GetClient(flags.kubeConfigPath, dryRun)
```

Comment:
I flip flopped on leaving a copy of `GetClient` on the front end. like I was leaving `enforceRequirements` but this seemed minor enough and left fewer copies of code around.